packages/samples/iife/build-assets.mjs

```diff
@@ -0,0 +1,120 @@
+import ncp from 'ncp';
+import {mkdirSync, readFileSync} from 'node:fs';
+import {dirname, join, relative, resolve as resolvePath} from 'node:path';
+import resolve from 'resolve';
+import {promisify} from 'util';
+import {workspacesRoot} from '../../../scripts/packages.mjs';
+
+const ncpAsync = promisify(ncp);
+
+/**
+ *
+ * @param {readonly V[]} values
+ * @param {(value: V, index: number) => K} getKey
+ * @template V
+ * @template {PropertyKey} K
+ * @returns {Record<K, V[]>}
+ */
+function aggregate(values, getKey) {
+  return values.reduce((aggregatedValues, value, i) => {
+    const key = getKey(value, i);
+    if (!(key in aggregatedValues)) {
+      aggregatedValues[key] = [];
+    }
+    aggregatedValues[key].push(value);
+    return aggregatedValues;
+  }, {});
+}
+
+/**
+ * @typedef CDNAsset
+ * @property {string} sourceDirectory Directory containing the required assets, relative to the root of the workspace.
+ * @property {string[]} relativeDestinations Where the directory needs to be accessible from. For example: `atomic/storybook`.
+ */
+
+/**
+ * @returns {CDNAsset[]}
+ */
+function getDeploymentPipelineAssets() {
```

Comment:
I decided to use the deployment pipeline config as a source of truth as to what gets exported to the CDN. This should help us catch errors.